pxtsim/simdriver.ts

```diff
@@ -690,7 +688,8 @@ namespace pxsim {
             msg.frameCounter = ++this.frameCounter;
             msg.options = {
                 theme: this.themes[this.nextFrameId++ % this.themes.length],
-                mpRole: /[\&\?]mp=(server|client)/i.exec(window.location.href)?.[1]?.toLowerCase()
+                mpRole: /[\&\?]mp=(server|client)/i.exec(window.location.href)?.[1]?.toLowerCase(),
+                hideSimButtons: !!/hidesimbuttons(?:[:=])1/i.exec(window.location.href)
```

Comment:
.test instead of exec?
```suggestion
                hideSimButtons: /hidesimbuttons(?:[:=])1/i.test(window.location.href)
```